lm-api/lm_api/security.py

```diff
@@ -17,33 +16,11 @@
 from lm_api.config import settings
 
 
-def get_domain_configs() -> typing.List[DomainConfig]:
-    domain_configs = [
-        DomainConfig(
-            domain=settings.ARMASEC_DOMAIN,
-            debug_logger=logger.debug if settings.ARMASEC_DEBUG else None,
-            use_https=settings.ARMASEC_USE_HTTPS,
-        )
-    ]
-    if all(
-        [
-            settings.ARMASEC_ADMIN_DOMAIN,
-            settings.ARMASEC_ADMIN_MATCH_KEY,
-            settings.ARMASEC_ADMIN_MATCH_VALUE,
-        ]
-    ):
-        domain_configs.append(
-            DomainConfig(
-                domain=settings.ARMASEC_ADMIN_DOMAIN,
-                match_keys={settings.ARMASEC_ADMIN_MATCH_KEY: settings.ARMASEC_ADMIN_MATCH_VALUE},
-                debug_logger=logger.debug if settings.ARMASEC_DEBUG else None,
-                use_https=settings.ARMASEC_USE_HTTPS,
-            )
-        )
-    return domain_configs
-
-
-guard = Armasec(domain_configs=get_domain_configs())
+guard = Armasec(
+    domain=settings.ARMASEC_DOMAIN,
+    debug_logger=logger.debug if settings.ARMASEC_DEBUG else None,
```

Comment:
you could do something like [this](https://github.com/omnivector-solutions/vantage-api/blob/main/vantage-api/api/settings.py#L123) so the possibilites of logging are increased

Reply:
the ability to customize the logging level. This can be helpful when debugging a particular issue